app/templates/partials/header.html

```diff
@@ -12,6 +12,13 @@
                     </div>
                 {% endblock logo %}
                 </div>
+                <div class="grid__col col-6@s">
+                    <nav class="nav nav--horizontal  u-d-no--@xs@s u-fr" aria-label="Site services menu">
+                        <ul class="nav__list" role="menubar" aria-label="Site services menu">
+                            <li class="nav__item" role="menuitem"><a href="{{respondent_account_url}}" id="my-account" class="nav__link">My account</a></li>
```

Comment:
You need to mark any new static content for translation. ie {{ _(My text) }}

Reply:
+ Generate new message.pot file